assets/semi-structured/clickhouse/docker-build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+container_name="clickhouse-clp-bench"
+
+docker build \
+    -t "$container_name" \
```

Comment:
```suggestion
    --tag "$container_name" \
```

Our convention nowadays is to use the full name for command line options since that's easier to read.